### PR TITLE
Add support for Terraform 0.11.11; remove oldest 0.9.x

### PR DESCRIPTION
**Pull request by hectcastro**
Add version 0.11.11 to the Travis CI build matrix. Remove the oldest from the 0.9.x series (this won't delete the existing container images on Quay).

See: https://travis-ci.org/azavea/docker-terraform/builds/475735659

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -21,4 +21,4 @@ deploy:
     script: "scripts/cipublish"
     on:
       repo: azavea/docker-terraform
-      branch: master
+      branch: hmc/terraform-0.11.11
```
Comment (hectcastro) [inline .travis.yml]: Reminder to revert this change before merge.

**Review by rbreslow — APPROVED** [review]
👍
